pages/collection.html

```diff
@@ -78,8 +78,22 @@ <h1>The Regis Altare Birthday Project 2023</h1>
       <section>
         <button id="full-collection-toggle">Show Full Collection</button>
         <button id="reset-collection">Reset Card Collection</button>
+        <label>
+            Sort by:
+            <select id="sort-dropdown">
+                <option value="Collector Number" selected>Collector Number</option>
+                <option value="Rarity">Rarity</option>
+                <option value="Rarity-">Rarity (Reverse)</option>
+            </select>
+        </label>
+
         <p id="collected-cards-number">Collected cards:</p>
         <section id="card-list"></section>
+        <div id="collection-pagination">
+          <a id="pagination-previous" href="javascript:void(0)" class="hidden"><</a>
+          <a id="pagination-current" href="javascript:void(0)">1</a>
```

Comment:
As a quick change suggestion, could we also show total pages, like "1 of 2"? The "of 2" can be plain text

As a bigger change I'm not sure we need to do, I'm thinking about how important it is to have the functionality to quickly find specific cards without having to click 'next page' a bunch of times. I wonder if it'd be possible to have a "cards per page" dropdown with (10, 20, 50, show all) or be able to jump to specific pages via url (collection.html/5)

Reply:
Quick change done. o7

The cards per page dropdown can be done. The url part is somewhat more complex to ensure they can't get into invalid pages and stuff, but it can be done too. I'll work on that next PR.